assets/meta.yaml

```diff
@@ -410,3 +410,190 @@
+    explanation: According to sections [[Segment Anything Dataset]](https://arxiv.org/pdf/2304.02643.pdf#section.5)
+      and [[Datasheet]](https://arxiv.org/pdf/2304.02643.pdf#page=25) of the paper.
+  # Downstream
+  access:
+    value: open
+    explanation: >
+      The full dataset can be downloaded at [[SA-1B Download]](https://ai.facebook.com/datasets/segment-anything-downloads/).
+      A 50k image preview of the full dataset is available [[here]](https://segment-anything.com/dataset/index.html).
+  intended_uses:
+    value: SA-1B is intended to be used for research purposes only. It allows access
+      to a privacy protecting and copyright friendly large-scale image dataset.
+      Researchers can use it to train and evaluate generic object segmentation models.
+    explanation: See [[SA-1B website]](https://ai.facebook.com/datasets/segment-anything/)
+  prohibited_uses:
+    value: >
+      Authors note the following limitations of the dataset:
+        The masks are generated by a segmentation model, so there may be errors
+      or inconsistencies in the masks.
+        While no two images are the same, there are instances of images of the same
+      subject taken close together in time.
+        The dataset contains scenes of protests, or other gatherings that may suggest
+      religious beliefs, political opinions or union memberships that may be offensive.
+
+    explanation: See [[Datasheet]](https://arxiv.org/pdf/2304.02643.pdf#page=25)
+  monitoring:
+    value: >
+      The dataset will be hosted at https://ai.facebook.com/datasets/segment-anything
+      and maintained by Meta AI.
+      "If a user observes objectionable image(s) in the dataset, we invite them
+      to report the image(s) at segment-anything at meta.com for removal"
+      "To aid reproducibility of research using SA-1B, the only updates (to the
+      dataset) will be to remove reported images."
+      "We encourage users to gather further annotations for SA-1B. Any users who
+      generate annotations will be liable for hosting and distributing their annotations."
+    explanation: See [[Datasheet]](https://arxiv.org/pdf/2304.02643.pdf#page=25)
+  feedback: Feedback can be given via the feedback form on their website [segment-anything.com](https://segment-anything.com/)
+    or by emailing at segment-anything at meta.com.
+
+- type: model
+  name: SAM
+  # General
+  organization: Meta
+  description: A foundation model for image segmentation.
+  created_date:
+    value: 2023-04-05
+    explanation: The date the [[Meta blog post]](https://ai.facebook.com/blog/segment-anything-foundation-model-image-segmentation/)
+      was released.
+  url: https://arxiv.org/pdf/2304.02643.pdf
+  model_card:
+    value: https://arxiv.org/pdf/2304.02643.pdf#page=28
+    explanation: Model card can be found in the Appendix section of the paper.
+  modality: image, segmentation prompt (point, box, mask and text)
```

Comment:
Unsure about this, should it just be image, text?

Reply:
i think image, text is fine. And the segmentation info could go into the description.